internal/kafka/test/mocks/kasfleetshardsync/kas-fleetshard-sync.go

```diff
@@ -48,27 +48,31 @@ var defaultUpdateDataplaneClusterStatusFunc mockKasFleetshardSyncUpdateDataPlane
 	}
 
 	for _, cluster := range clusters {
-		managedKafkaAddon, err := ocmClient.GetAddon(cluster.ClusterID, ocmConfig.StrimziOperatorAddonID)
-		if err != nil {
-			return err
-		}
-
-		kasFleetShardOperatorAddon, err := ocmClient.GetAddon(cluster.ClusterID, ocmConfig.KasFleetshardAddonID)
-		if err != nil {
-			return err
-		}
+		if cluster.ProviderType != api.ClusterProviderStandalone {
+			managedKafkaAddon, err := ocmClient.GetAddon(cluster.ClusterID, ocmConfig.StrimziOperatorAddonID)
+			if err != nil {
+				return err
+			}
 
-		if managedKafkaAddon.State() == clustersmgmtv1.AddOnInstallationStateReady && (kasFleetShardOperatorAddon.State() == clustersmgmtv1.AddOnInstallationStateReady || kasFleetShardOperatorAddon.State() == clustersmgmtv1.AddOnInstallationStateInstalling) {
-			ctx, err := NewAuthenticatedContextForDataPlaneCluster(helper, cluster.ClusterID)
+			kasFleetShardOperatorAddon, err := ocmClient.GetAddon(cluster.ClusterID, ocmConfig.KasFleetshardAddonID)
 			if err != nil {
 				return err
 			}
 
-			clusterStatusUpdateRequest := SampleDataPlaneclusterStatusRequestWithAvailableCapacity()
-			if _, err := privateClient.AgentClustersApi.UpdateAgentClusterStatus(ctx, cluster.ClusterID, *clusterStatusUpdateRequest); err != nil {
-				return fmt.Errorf("failed to update cluster status via agent endpoint: %v", err)
+			if !(managedKafkaAddon.State() == clustersmgmtv1.AddOnInstallationStateReady && (kasFleetShardOperatorAddon.State() == clustersmgmtv1.AddOnInstallationStateReady || kasFleetShardOperatorAddon.State() == clustersmgmtv1.AddOnInstallationStateInstalling)) {
+				return nil
```

Comment:
Why are returning here? maybe `continue` instead
```suggestion
				continue
```